scraper/src/tests/default_strategy/get_anchor_test.py

```diff
@@ -107,3 +107,45 @@ def test_no_anchor(self):
 
         # Then
         assert actual is None
+
+    def test_linking_anchor(self):
+        # Given
+        strategy = get_strategy()
+        strategy.dom = lxml.html.fromstring("""
+        <html><body>
+            <h1>Foo</h1>
+            <h2 id='_docusaurus'>Bar</h2>
```

Comment:
The actual Docusaurus anchor is `__docusaurus` (2 underscores), Gatsby's is `___gatsby` (3 underscores). If we want to be safe that it doesn't skip anchors that were originally scraped, we might want to skip the anchor only if it starts with two underscores.

Can we do a pass on multiple DocSearch indices to check that people's websites don't use anchors prefixed by `_`?

Reply:
It sounds pretty weird to use such kind of anchor. I have updated it to make sure we check for both 2 underscore. I have tested it and it seems fine. I do not think this feature require a specific setting. I will add it as the default behaviour